src/serac/physics/nonlinear_solid.cpp

```diff
@@ -118,6 +119,11 @@ void NonlinearSolid::setTractionBCs(const std::set<int>&                     tra
   bcs_.addNatural(trac_bdr, trac_bdr_coef, component);
 }
 
+void NonlinearSolid::setExternalForces(std::vector<std::shared_ptr<mfem::VectorCoefficient>> ext_force_coefs)
```

Comment:
Who are these vector coefficients shared with?

Reply:
The vector coefficients might be created by the user (or sidre) and shared with the `NonlinearSolid`. The user or another solver might change the vector coefficients over time. 

This was one way to guarantee the VectorCoefficient's don't get out of scope. It's similar to the displacement and tractionBCs.

Reply:
I know these aren't really boundary conditions, but it does describe an external loading and a lot of production codes treat it as such. Should we add this as another boundary condition type? Any thoughts @samuelpmishLLNL @joshessman-llnl @solberg2 ?

Reply:
These are body forces, I think?  So yes it makes sense to treat them as a boundary condition.

Sometimes people will have body forces coming from an external source, basically electromagnetic forces.

Sometimes people will have body forces coming from an imposed external acceleration (gravity, centripetal acceleration)

Sometimes people will have body forces coming from the method of manufactured solutions.